Source/Dafny/Verifier/Translator.cs

```diff
@@ -700,6 +701,40 @@ internal class PredefinedDecls {
       return new Bpl.IdentifierExpr(tok, var.AssignUniqueName(currentDeclaration.IdGenerator), TrType(var.Type));
     }
 
+    private void AddBitwidth(int w) {
+      if (generatedBitwidths.Contains(w)) {
```

Comment:
Consider writing this as `if (!generatedBitwidths.Add(w)) { return; }` and dropping the `generatedBitwidths.Add(w);` at the end of this method.